src/Components/Button/index.js

```diff
@@ -4,7 +4,7 @@ import "./index.css";
 export default function index(props) {
   return (
     <div>
-      <button className='contact-right'>{props.name} </button>
```

Comment:
For now this is ok, but in case there will be more buttons, then it would be nice to be able to use this styling but then with different names.